DietPi-Software | Emby-server: Install latest version via "api.github.com"

MichaIng opened the pull request:
**Status**: Ready
- [x] Disable for ARM+Jessie (see link below)
- [x] Changelog
- [x] https://github.com/Fourdee/DietPi/wiki/DietPi-Software-list update: https://github.com/Fourdee/DietPi/wiki/_compare/2e14eccade9b32b88d49b318bf0041fab1f5c09f

**Testing**:
- [x] Jessie VM (x86_64, install, service state, web UI)
- [x] Stretch VM (install, service state, web UI)
- [x] Buster VM (install, service state, web UI)
- It seems like there is an internal updater available anyway? In this case skip `deb` reinstall, if already there and leave update to user via web interface, which should be safer in terms of database migration and stuff?

**Reference**: https://github.com/Fourdee/DietPi/issues/2521

**Commit list/description**:
+ DietPi-Software | Emby-server: Install latest version via "api.github.com", which offers native ARMv8 packages as well

MichaIng committed:
v6.22

+ DietPi-Software | Emby-server: Install latest version via "api.github.com", which offers native ARMv8 packages as well

MichaIng committed:
v6.22

+ DietPi-Software | Emby-server: Whoopsie

MichaIng committed:
v6.22

+ CHANGELOG | Emby server: Now installs the latest version automatically
+ CHANGELOG | Remove doubled "additional issues" entry. Keep this only in most current version, same as "Known/Outstanding Issues".

Fourdee committed:
Merge branch 'dev' into emby_update

Fourdee reviewed (approved):
@MichaIng 

Many thanks, looking good 👍 

Please merge when ready.

MichaIng commented:
@Fourdee 
As of referenced issue: Shall we create own Jessie compatible ARM binaries? Would be additional work for you mainly since I am not experienced with source builds and lack devices for all archs respectively cross-compile knowledge.

Otherwise, more likely (?), I will add dropping Jessie support for Emby server, but we then really need to provide an Odroid C1 Stretch image before dropping more and more support for it. Could be up to 400 active users, when counting survey opted-outs as well.

MichaIng committed:
v6.22

+ DietPi-Software | Emby Server: Drop ARM support on Jessie due to libc6 version requirement

MichaIng committed:
v6.22

+ CHANGELOG | Emby Server: Add info about disabled install option on Jessie ARM systems

MichaIng commented:
Okay, Jessie + ARM support dropped for now. Will merge.
